restcomm.android.client.sdk/src/main/java/org/mobicents/restcomm/android/client/sdk/RCConnectionListener.java

```diff
@@ -78,5 +78,12 @@
      * @param videoTrack Video track hosting the remote video
      */
     public void onReceiveRemoteVideo(RCConnection connection, VideoTrack videoTrack);
+
+    /**
+     * Called in case of Android permission error.
+     * @param connection  Connection
+     * @param permission The permission that is not granted.
+     */
+    public void onPermissionNotGranted(RCConnection connection, String permission);
```

Comment:
@mehturt do you still need this? I thought we talked about using existing error reporting callbacks. Am I missing sth?